### DISCUSSION TITLE
Easier way to copy the address

### DISCUSSION THREAD

**Discussion by HubKing** [blueman-project/blueman#2235] [Ideas]
How about the context menu having a "copy address" item or making it copy the address to press ctrl+C when a device is selected in the main window?

**Comment by cschramm** [comment]
What do you regularly need it for?

**Reply by HubKing** [reply]
So that I could use the address in scripts or other command-line apps. In my case, I was trying to copy the addresses of multiple Bluetooth thermometers I have, and pass the addresses to a script.
